src/database/utilities/constants.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/prefer-default-export */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+export const MILLISECONDS_PER_DAY = 24 * 60 * 60 * 1000;
```

Comment:
Going to make a potentially controversial suggestion here:

```
const HOURS_PER_DAY = 24;
const MINUTES_PER_HOUR = 60;
const SECONDS_PER_MINUTE = 60;
const MILLISECONDS_PER_MINUTE = 1000;

const MILLISECONDS_PER_MINUTE = MILLISECONDS_PER_SECOND * SECONDS_PER_MINUTE
const MILLISECONDS_PER_HOUR = MILLISECONDS_PER_MINUTE * MINUTES_PER_HOUR
const MILLISECONDS_PER_DAY = MILLISECONDS_PER_HOUR * HOURS_PER_DAY
```

Reply:
Thanks for starting WW3

Reply:
23a081c2ad1a430e53100d870fe5ee12f248c634 thanks good idea